packet/Cargo.toml

```
@@ -35,8 +35,10 @@ serde_derive = { workspace = true, optional = true }
 serde_with = { workspace = true, optional = true, features = ["macros"] }
 solana-frozen-abi = { workspace = true, optional = true, features = ["frozen-abi"] }
 solana-frozen-abi-macro = { workspace = true, optional = true, features = ["frozen-abi"] }
+solana-pubkey = { workspace = true, features = ["serde"] }
 
 [dev-dependencies]
+solana-address = { workspace = true, features = ["atomic"] }
```

nit: it's a bit confusing to mix address and pubkey, so probably clearer to use the `std` feature on `solana-pubkey`
```suggestion
solana-pubkey = { workspace = true, features = ["std"] }
```